refinery/file_store/test_models.py

```diff
@@ -220,26 +216,17 @@ def test_set_local_file_type_update(self):
         saved_item = FileStoreItem.objects.get(pk=self.item.pk)
         self.assertEqual(saved_item.filetype, zip_file_type)
 
-    @override_storage()
     def test_get_local_file_extension(self):
         self.item.datafile.save(self.file_name, ContentFile(''))
         saved_item = FileStoreItem.objects.get(pk=self.item.pk)
         self.assertEqual(saved_item.get_file_extension(), self.file_extension)
 
-    @override_storage()
     def test_delete_local_file_on_instance_delete(self):
         self.item.datafile.save(self.file_name, ContentFile(''))
-        with mock.patch.object(FieldFile, 'delete') as mock_delete:
-            self.item.delete()
-            mock_delete.assert_called_with(save=False)
-
-    @override_storage()
-    def test_delete_local_file_on_instance_update(self):
```

Comment:
Is this a behavior that we don't want to test anymore?

Reply:
Correct.